docs/pages/access-controls/device-trust/enforcing-device-trust.mdx

```diff
@@ -199,8 +200,58 @@ version: v2
 ```
 
 Now the alice user can only access `env:production` apps using a trusted device.
-For example, to access an app called `myapp`, alice runs `tsh proxy app myapp -p
-8888` and opens http://localhost:8888 in her browser.
+
+## Desktop Access support
+
+Desktop access may enforce device trust via [role-based enforcement](
+#role-based-trusted-device-enforcement).
+
+To access desktops protected by device trust make sure your device is
```

Comment:
Are we capitalizing "device trust"? I would use consistent capitalization so readers know whether to treat this is as a Teleport-specific feature—if it's a general computing term that's not unique to Teleport, it would make sense to use lowercase everywhere.

Reply:
I'll take a look and see what we are doing. Do you have a preference for capitalized or not in this particular case?

Reply:
Sorry for the delay. It's a mixed bag for sure, we use both capitalized and not in various places. If you have a suggestion, @ptgott, I'd be happy to write a follow-up "normalizing" it.

Reply:
The AWS Verified Access docs [say "device trust"](https://docs.aws.amazon.com/verified-access/latest/ug/device-trust.html), although Okta [capitalizes "Device Trust"](https://help.okta.com/en-us/content/topics/mobile/okta_mobile_device_trust_windows-desktop.htm). I think this is a general enough computing term that "device trust" makes sense, though to be honest I can see "Device Trust" making sense as well. I'll ask Marketing and write a Vale rule for it so we can decouple the standardization work from this PR!

Reply:
Sounds great, thanks!